src/audio/base_fw.c

```diff
@@ -391,6 +392,35 @@ static int basefw_power_state_info_get(uint32_t *data_offset, char *data)
 	return 0;
 }
 
+static int fw_config_set_force_l1_exit(const struct sof_tlv *tlv)
+{
+	const uint32_t force = tlv->value[0];
+
+	if (force)
+		pm_device_force_dmi_l0_state();
+	else
+		pm_device_allow_dmi_l1_state();
+
+	return 0;
+}
+
+static int basefw_set_fw_config(bool first_block,
+				bool last_block,
+				uint32_t data_offset,
+				const char *data)
+{
+	const struct sof_tlv *tlv = (const struct sof_tlv *)data;
+
+	switch (tlv->type) {
+	case IPC4_DMI_FORCE_L1_EXIT:
+		return fw_config_set_force_l1_exit(tlv);
```

Comment:
It wouldn't hurt to describe when host software is supposed to send this. My understanding of L1 is that this is a link/DMA state that has very large impacts on platform power consumption - or rather than staying in L0 can be problematic.

To the best of my knowledge this isn't supported today in the Linux driver.

@kv2019i is this on your radar?

Reply:
@fkwasowi can you add an entry into the logs when host enables/disables this features to help anyone debugging FW issues.

Reply:
I added log info and a more complete description of the PR change.